runtime/src/iree/schemas/bytecode_module_def.fbs

```diff
@@ -246,7 +246,7 @@ table BytecodeModuleDef {
   // Features required to load and run bytecode from the module.
   // Some functions may have additional requirements for when there are multiple
   // variants selected based on runtime feature detection.
-  requirements:FeatureBits = 0;
+  requirements:FeatureBits = EXT_32;
```

Comment:
From the presubmit failures, it looks like this may indeed be a version mismatch. We may need to carry a temporary patch google3-side while we work out which flatcc version needs updating. One interesting thing, the error is:

> 2023-04-13T17:43:42.9476253Z bytecode_module_def.fbs:249:30: error: 'EXT_3': unknown name used as initializer for enum field: bytecode_module_def.fbs:249:3: 'requirements'

So `EXT_3` not `EXT_32`

Reply:
I updated our downstream flatcc version, so this might not be needed.

Reply:
Ah looks like the presubmit failures were because of a typo. Passing now. So regardless of what errors where, it seems like referencing the enum by name would be better (and consistent with the rest of the file) anyway. @benvanik is there some reason for this not to be the enum name?